refarch-apigateway/frontend/src/plugins/vuetify.ts

```diff
@@ -0,0 +1,20 @@
+import "@mdi/font/css/materialdesignicons.css";
+import "vuetify/styles";
+
+import { createVuetify } from "vuetify";
+
+export default createVuetify({
+  theme: {
+    themes: {
+      light: {
+        colors: {
+          primary: "#333333",
+          secondary: "#FFCC00",
+          accent: "#7BA4D9",
+          success: "#69BE28",
+          error: "#FF0000",
```

Comment:
Maybe we should make the colors configurable via the initialization tool? Just an idea.. Also, dark mode needs to be supported..

Reply:
I think we should actually dont make it configurable, because all application should have a consistent look. If someone wants to change it, then he should do it manually. But yeah dark-mode is missing. For the dark-mode i added #42

Reply:
Ok! Fine! 😄